roxie/udplib/udptrs.cpp

```diff
@@ -785,7 +785,7 @@ class CSendManager : public CInterface, implements ISendManager
             {
                 if (!sniffer_socket) 
                 {
-                    sniffer_socket = ISocket::multicast_connect(ep, 3);
+                    sniffer_socket = ISocket::multicast_connect(ep, multicastTTL);
```

Comment:
This is not the only multicast connection that Roxie uses - should it also affect the others?